src/library/Uno.Material/Styles/Controls/ToggleSwitch.xaml

```diff
@@ -23,7 +23,7 @@
 
 		<!-- microsoft/microsoft-ui-xaml#6157: reset min-width inherited from base style -->
```

Comment:
Does the comment still apply ?

Reply:
The issue still remains opened. I've also checked, on both latest stable and latest prerelease, the ToggleSwitch still have that MinWidth hardcoded.